sphinx/tutorial/rose/furthertopics/macro-development.rst

```diff
@@ -90,16 +90,20 @@ Open ``planet.py`` in a text editor and paste in the following code:
 
 .. code-block:: python
 
-   #!/usr/bin/env python2
-   # -*- coding: utf-8 -*-
+   #!/usr/bin/env python3
```

Comment:
Could do with a note explaining that this is python2/3 compatible code in order to be rose1/2 compatible in order to use the rose 1 Gui.

I'm not sure whether this shebang is needed? Does the tutorial execute this file `chmod +x <file>; ./file`, if not can remove.